sparkler-app/src/main/scala/edu/usc/irds/sparkler/pipeline/Crawler.scala

```diff
@@ -84,6 +87,9 @@ class Crawler extends CliTool {
     usage = "Delay between two fetch requests")
   var fetchDelay: Long = sparklerConf.get(Constants.key.FETCHER_SERVER_DELAY).asInstanceOf[Number].longValue()
 
+  @Option(name = "-aj", aliases = Array("--add-jar"), usage = "Add sparkler jar to spark context")
+  var path : String = ""
```

Comment:
Let's make this a generic solution which accepts a list of jars.
Please refer this - http://args4j.kohsuke.org/apidocs/org/kohsuke/args4j/spi/StringArrayOptionHandler.html
Also, it would be nice to use the variable name `jarPath` instead of `path` ?